### PR TITLE
FLOE-524: Add a News entry on the Science Lab

### PR DESCRIPTION
**Pull request by simonbates**

### CODE REVIEW

File: news/2017-07-19-inclusive-science-lab.html [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+            <div class="floe-content floe-news-item">
+                <h2> News </h2>
+
+                <h3>Inclusive Science Lab</h3>
+                <p class="floe-date">July 19, 2017</p>
+                <p>
+                    We've been building an integration technology called the Nexus. It allows the interconnection of arbitrary devices by co-ordinating device state over HTTP and WebSockets.
+                </p>
+                <p>
+                    Recently, we have been using it to try out designs to make science labs more inclusive. The Nexus is used to connect lab sensors, such as pH sensors and electrical conductivity sensors, to a range of different presentations and interactions. So that a student can pick the interaction that works best for them. For some students this might be a visualization and for others it might be a sonification, or a combination of different presentations.
```
Comment (colinbdclark) [inline news/2017-07-19-inclusive-science-lab.html]: There's a funny kind of sentence fragment here: "So that a student can pick the interaction that works best for them." Perhaps "This helps a student pick the interaction that works best for them" or something like this?

File: index.html [hunk 2]
```diff
@@ -251,6 +251,12 @@
                         <div>
                             <div class="floe-front-news">
                                 <h3>News</h3>
+
+                                <h4>Inclusive Science Lab</h4>
+                                <p class="floe-date">July 19, 2017</p>
+                                <p>We've been working recently on designs to make science labs more inclusive. We recently invited two students to try out some things that we have built and to make a video with us.</p>
```
Comment (colinbdclark) [inline index.html]: What a nice and succinct summary!

Comment (jhung) [inline index.html]: I think this opening could be massaged a bit to read a bit better. Maybe:
"We've been working on designs to make science labs more inclusive, and invited two students to try some things we built. They also helped us make a video!"

File: news/2017-07-19-inclusive-science-lab.html [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+            </header>
+
+            <div class="flc-toc-tocContainer toc"> </div>
+            <div class="floe-content floe-news-item">
+                <h2> News </h2>
+
+                <h3>Inclusive Science Lab</h3>
+                <p class="floe-date">July 19, 2017</p>
+                <p>
+                    We've been building an integration technology called the Nexus. It allows the interconnection of arbitrary devices by co-ordinating device state over HTTP and WebSockets.
```
Comment (jhung) [inline news/2017-07-19-inclusive-science-lab.html]: Can we add an appropriate link for the Nexus?

File: news/2017-07-19-inclusive-science-lab.html [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+                </p>
+                <p>
+                    Recently, we have been using it to try out designs to make science labs more inclusive. The Nexus is used to connect lab sensors, such as pH sensors and electrical conductivity sensors, to a range of different presentations and interactions. This helps a student pick the interaction that works best for them. For some students this might be a visualization and for others it might be a sonification, or a combination of different presentations.
```
Comment (jhung) [inline news/2017-07-19-inclusive-science-lab.html]: Ambiguous as to what "it" refers to. Suggest rewording: 
"Using the Nexus we have been exploring designs to make science labs more inclusive..."

File: news/2017-07-19-inclusive-science-lab.html [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+                </p>
+                <p>
+                    We invited Calum and Miles to try out an inclusive science lab design, and to help us make a video about it:
```
Comment (jhung) [inline news/2017-07-19-inclusive-science-lab.html]: Suggest re-wording:

"Calum and Miles joined us to try out an inclusive science lab design, and helped us make a video about their experiences:"

**Comment by jhung** [comment]
Any changes to the 2017-07-19-inclusive-science-lab.html  should also be replicated in the News archive index.html.

**Comment by simonbates** [comment]
Thanks for the suggestions @jhung -- I've updated the pull request.
